runtime/doc/api.txt

```diff
@@ -3042,6 +3042,12 @@ nvim_open_win({buffer}, {enter}, {*config})                  *nvim_open_win()*
                       specified by character: [ {"+", "MyCorner"}, {"x",
                       "MyBorder"} ].
 
+                  • title: Title (optional) in window border, keys
+                    • "title_text": text in borer.
+                    • "title_pos": the position of title. String value can be
+                      left, center, right.
+                    • "title_hl": higlight group of title.
```

Comment:
Why not use hl group for this? i.e `NormalFloatTitle` or something.

Reply:
by default is FloatBorderTitle. also support you custom it for every floatwin. there only need write test for it so it's draft.